tests/konflux-demo/config/scenarios.go

```diff
@@ -31,7 +23,7 @@ var TestScenarios = []TestSpec{
 				DockerFilePath:             "Dockerfile",
 				HealthEndpoint:             "/",
 				SkipDeploymentCheck:        true,
-				AdvancedBuildSpec: &AdvancedBuildSpec{
+				BuildSpec: &BuildSpec{
```

Comment:
since we're no longer distinguishing between advanced/simple build, would it make sense to remove the wrapping struct `BuildSpec` and keep only `TestScenario` (and maybe rename it to IntegrationTestScenario`?

Reply:
good point, should we then also remove `SkipDeploymentCheck` and `HealthEndpoint` fields? since they're not used anywhere?

Reply:
yes, that definitely makes sense

Reply:
removed `BuildSpec` and replaced it with `IntegrationTestScenario` struct. also removed SkipDeploymentCheck and HealthEndpoint fields. thanks!